src/helper/enumerate-entries.ts

```diff
@@ -77,16 +77,59 @@ export function referenceToHTML(
   renderOption: RenderOption,
   renderEmbed?: (metadata: Metadata) => EmbeddedItem | EntryNode,
 ): string {
+  if ( node.children!== undefined && node.attrs.type === 'asset' && node.attrs['content-type-uid'] === 'sys_assets') {
+    
+    // Extract image information
+    const src = node.attrs['asset-link'];
+    const alt = node.attrs?.['redactor-attributes']?.['alt'];
+    const link = node.attrs.link;
+    const target = node.attrs.target || "";
+    const caption = node.attrs?.['redactor-attributes']?.['asset-caption'] || node.attrs?.['asset-caption'] || "";
+    const style = node.attrs.style;
+    const asset_uid= node.attrs['asset-uid'];
```

Comment:
@reeshika-h , [Minor] Used destructuring to extract values from node.attrs for better readability.
  const { 
    'asset-link': src, 
    link, 
    target = "", 
    style, 
    'asset-uid': assetUid,
    'redactor-attributes': redactorAttributes = {},
    'asset-caption': assetCaption 
  } = node.attrs;